packages/mc-scripts/src/commands/build-vite.ts

```diff
@@ -26,13 +27,13 @@ async function run() {
 
   await build({
     root: paths.appRoot,
+    base: './', // <-- Important to allow configuring the runtime base path.
     define: {
       'process.env.DEBUG': JSON.stringify(false),
       'process.env.NODE_ENV': JSON.stringify('production'),
     },
     build: {
       outDir: 'public',
-      assetsDir: '.',
```

Comment:
Currently we have all public files (`index.html`, favicons, etc.) AND the assets files (JS / CSS chunks) as a flat list inside the `public` folder.

I don't see any specific benefit from doing that (it was kind of a back and forth with some previous changes) so I think at least for Vite we can stick with the default behavior, which is having an `assets` folder.

This would make it easier to move this `assets` folder to a CDN or something, while leaving the other "public" files inside the `public` root folder.

<img width="207" alt="image" src="https://user-images.githubusercontent.com/1110551/179011571-c0827f7b-0c1c-4d23-b72c-ae9ac3de9d8d.png">